gui/NumeReWindow.cpp

```diff
@@ -312,7 +312,7 @@ bool MyApp::OnInit()
     // Create and initialize the main frame. Will also
     // include loading the configuration, loading existing
     // caches and preparing the editor.
-    NumeReWindow* NumeReMainFrame = new NumeReWindow("NumeRe: Framework f�r Numerische Rechnungen (v" + sVersion + ")", wxDefaultPosition, wxDefaultSize);
+    NumeReWindow* NumeReMainFrame = new NumeReWindow("NumeRe: Framework für Numerische Rechnungen (v" + sVersion + ")", wxDefaultPosition, wxDefaultSize);

```

Comment:
Careful with changes in the encoding. Might break things, we'll see.

Reply:
![grafik](https://user-images.githubusercontent.com/87755490/235253705-80c97b94-9042-4217-b805-77044673f58b.png)
Did break something